ALUM/ALUM/Views/MenteeProfileScreen.swift

```diff
@@ -115,43 +62,125 @@ struct MenteeProfileScreen: View {
                 .edgesIgnoringSafeArea(.bottom)
             }
             ZStack {
-                if !viewModel.selfView! {
-                    // params currently placeholders for later navigation
-                    if scrollAtTop {
-                        NavigationHeaderComponent(
-                            backText: "Login",
-                            backDestination: LoginScreen(),
-                            title: "Mentee Profile",
-                            purple: true,
-                            showButton: false
-                        )
-                        .background(Color("ALUM Primary Purple"))
-                    } else {
-                        NavigationHeaderComponent(
-                            backText: "Login",
-                            backDestination: LoginScreen(),
-                            title: "Mentee Profile",
-                            purple: false,
-                            showButton: false
-                        )
-                        .background(.white)
-                    }
-                } else {
-                    if scrollAtTop {
-                        ProfileHeaderComponent(profile: true, title: "My Profile", purple: true)
-                            .background(Color("ALUM Primary Purple"))
-                    } else {
-                        ProfileHeaderComponent(profile: true, title: "My Profile", purple: false)
-                            .background(.white)
-                    }
+                // Removing this Z-stack causes a white rectangle to appear between the top of screen 
+                // and start of this screen due to GeometryReader
+                
+                if viewModel.selfView! {
+                  // params like settings and edit profile currently placeholders for later navigation
+                  if scrollAtTop {
+                    ProfileHeaderComponent(profile: true, title: "My Profile", purple: true)
```

Comment:
@petabite Routing is finalized now. Buttons like edit profile and settings can just be added to `ProfileHeaderComponent`

We don't need to use the navbar with the back button for this case because mentor/mentee profile screens will only have the edit and settings icon when the someone views their own profile which is only possible when they get to the profile screen through the tab view. That's why if you observer the custom navbar is hidden n the ProfileTabRouter